### ISSUE TITLE
jx create devpod - should automatically detect the container type

### ISSUE THREAD

**Issue by michaelneale** [jenkins-x/jx#767]
based on the current directory. So if you have imported a project, you can just open a devpod right from its dir - and nothing else to do

**Comment by jstrachan** [comment]
Agreed - it can detect the `Jenkinsfile` and find the agent name from that

**Comment by michaelneale** [comment]
Or it could use the draft stuff to detect? Presumably it is already setup in Jenkins x so maybe there is some other bit of data it could use to detect? (Jenkinsfile could work for sure)

**Comment by jstrachan** [comment]
Agreed
